discretisedfield/plotting/mpl_region.py

```diff
@@ -114,5 +116,7 @@ def _setup_multiplier(self, multiplier):
         return self.region.multiplier if multiplier is None else multiplier
 
     def _axis_labels(self, ax, multiplier):
-        units = rf" ({uu.rsi_prefixes[multiplier]}{self.region.units})"
-        ax.set(xlabel=f"x {units[0]}", ylabel=f"y {units[1]}", zlabel=f"z {units[2]}")
+        unit_x = rf" ({uu.rsi_prefixes[multiplier]}{self.region.units[0]})"
+        unit_y = rf" ({uu.rsi_prefixes[multiplier]}{self.region.units[1]})"
+        unit_z = rf" ({uu.rsi_prefixes[multiplier]}{self.region.units[2]})"
+        ax.set(xlabel=f"x {unit_x}", ylabel=f"y {unit_y}", zlabel=f"z {unit_z}")
```

Comment:
We have (and had before) two spaces between the axis name and the unit. Any idea if this is intentional?

Reply:
I have no idea if this is intentional or not.

Reply:
Then I would say we remove it.
```suggestion
        unit_x = rf"({uu.rsi_prefixes[multiplier]}{self.region.units[0]})"
        unit_y = rf"({uu.rsi_prefixes[multiplier]}{self.region.units[1]})"
        unit_z = rf"({uu.rsi_prefixes[multiplier]}{self.region.units[2]})"
        ax.set(xlabel=f"x {unit_x}", ylabel=f"y {unit_y}", zlabel=f"z {unit_z}")
```